src/randoop/operation/ConstructorSignatures.java

```diff
@@ -0,0 +1,73 @@
+package randoop.operation;
+
+import java.lang.reflect.Constructor;
+
+/**
+ * ConstructorParser provides static methods to write as well as parse 
+ * a string representation of a constructor signature.
+ */
+public class ConstructorSignatures {
+
+  /**
+   * Parses a constructor signature as produced by 
+   * {@link ConstructorSignatures#getSignatureString(Constructor)} 
+   * and returns the corresponding reflective {@link Constructor} object.
+   * 
+   * @param signature a string representing a constructor signature.
+   * @return reflective {@link Constructor} method corresponding to signature.
+   * @throws OperationParseException if signature parameter does not match 
```

Comment:
This doesn't look right.  For example, it might throw an AssertionException if the format is wrong and one of the assertions fails.